sql-plugin/src/main/scala/org/apache/spark/sql/rapids/ColumnVectorUtil.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2020-2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.rapids
+
+import ai.rapids.cudf.ColumnVector
+import com.nvidia.spark.rapids.{Arm, GpuScalar}
+
+object ColumnVectorUtil extends Arm {
```

Comment:
nit: Could we work on naming this better? I don't really want a new `Util` that ends up being a dumping ground for random operations. This could go into GpuMapUtils if it is really specific to maps, or it could be renamed into GpuErrorUtils. If it is GpuErrorUtils, then I would prefer to see it pull in more and not just return a String, but actually throw the exception.

Reply:
Putting it in `GpuMapUtils` makes more sense because this is map specific and we don't have to add an additional utility class